vita3k/gui/src/live_area.cpp

```diff
@@ -1057,10 +1057,17 @@ void draw_live_area_screen(GuiState &gui, EmuEnvState &emuenv) {
             ImGui::Spacing();
             ImGui::SetCursorPosX(ImGui::GetWindowWidth() / 2.f - (ImGui::CalcTextSize("Using configuration set for keyboard in control setting").x / 2.f));
             ImGui::TextColored(GUI_COLOR_TEXT, "Using configuration set for keyboard in control setting");
-            if (gui.modules.empty()) {
+            const auto FW_PATH{ fs::path(emuenv.pref_path) / "vs0" };
+            const auto FW_FONT_PATH{ fs::path(emuenv.pref_path) / "sa0" };
+            if (fs::exists(FW_PATH) && fs::is_empty(FW_PATH)) {
```

Comment:
```suggestion
    if (gui.modules.empty()) {
```
keep it

Reply:
The text is displayed without loading the module even if the firmware is installed

Reply:
> The text is displayed without loading the module even if the firmware is installed

only module, almost files does not unused and work in vita3k.

Reply:
??
My point is that the module is not always loaded while the emulator is running.

Reply:
> ?? My point is that the module is not always loaded while the emulator is running.

but load modules in some games are needed.

Reply:
I repeat your carrément code is wrong, your if condition is wrong
And so no need check folder